website/content/v1.2/reference/api.md

```diff
@@ -24,6 +24,147 @@ description: Talos gRPC API reference.
   
     - [File-level Extensions](#common/common.proto-extensions)
   
+- [resource/definitions/cluster/cluster.proto](#resource/definitions/cluster/cluster.proto)
```

Comment:
I guess this should be reverted (changes only for 1.3)?

Reply:
What do you mean by reverted? This is 1.3 doc generated, so shouldn't this be included?

Reply:
I mean we shouldn't touch 1.2 docs anymore. 1.2 docs should be frozen until we decide to do some form of the backport.

Reply:
Oh, right! I removed leftovers.